src/main/java/org/opensearch/security/resolver/IndexResolverReplacer.java

```diff
@@ -697,7 +698,9 @@ private boolean getOrReplaceAllIndices(final Object request, final IndicesProvid
             //do nothing
         } else if (request instanceof SearchScrollRequest) {
             //do nothing
-        } else {
+        } else if (request instanceof PutComponentTemplateAction.Request) {
+            // do nothing
+        }else {
```

Comment:
nitpick: space before `else` (do we have a linting rule for this?)

Reply:
done